### PR TITLE
Added first requirements

### PR DESCRIPTION
**Pull request by vladdord**

### CODE REVIEW

File: Requirements.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ 3. System is able to store historical data
+
+## Plans for future
+
+ 1. Implementation real-time data obtaining once a second
+ 2. Implementation of prediction engine
+ 3. Implementation of new database architecture with unique timestamps and three tables for predicted, historical and real-time data
+ 4. Redesign of UI for more convenient usage
+
+## Restrictions
```
Comment (bedrin) [inline Requirements.md]: Should go to non-functional requirements section

File: Requirements.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    * [Intrinio](https://intrinio.com/)
+    * etc...
+
+## Current state
+
+ 1. System is able to obtain historical and real-time data with less frequency then required
+ 2. Prediction is not fully implemented
+ 3. System is able to store historical data
+
+## Plans for future
```
Comment (bedrin) [inline Requirements.md]: Remove it please - better create an issue ( https://github.com/RomanEgel/DeutscheBankProject/issues ) for each of them to track progress

All of them are already covered in requirements above (if not, it should go to requirements)

File: Requirements.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Non-Functional requirements
+
+ 1. Project must be able to process data fast and accurate
+ 2. UI must be convenient for end-user
+ 3. Data can be obtained from different sources
+    * [X-markets](https://www.xmarkets.db.com/DE/ENG/Underlying-Detail/XC0009677409)
+    * [Intrinio](https://intrinio.com/)
+    * etc...
+
+## Current state
```
Comment (bedrin) [inline Requirements.md]: These items also should go to functional or non-functional requirements above

File: Requirements.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+## Functional requirements
+
+ 1. Project must maintain [WebSocket](https://www.websocket.org/) technology for permanent connection and displaying real-time prices
+ 2. Store historical and real-time data in database(current is [MySQL](https://www.mysql.com/))
+ 3. Ability to predict Brent oil prices with machine learning algorithms(Self-wrietten algorithms and libraries for example [Spark MLLib](https://spark.apache.org/docs/latest/index.html))
+ 4. REST-API built on [Spring](https://spring.io/) engine
+
+## Non-Functional requirements
+
+ 1. Project must be able to process data fast and accurate
```
Comment (bedrin) [inline Requirements.md]: We need numbers here. What is fast? sub-second? few seconds?

File: Requirements.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    * predicted prices in user-defined date scope
+    * real-time price ticker with date and time for last price
+    * greetings page explining project goals and navigation bar through all content
+ 3. Project consists of several parts with different functionalities. At least:
+    * [REST API](https://en.wikipedia.org/wiki/Representational_state_transfer) for collecting whole project functionalities in one control panel
+    * Database for storing all predicted, real-time and historical data
+    * Data collector for real-time and historical data
+    * Predictor
+
+## Functional requirements
```
Comment (bedrin) [inline Requirements.md]: technical stuff (websokets, rest, spring, etc.) shouldn't be here - spring is not a requirement.
Suggest moving it to the bottom of the page to a new section called "Architecture" or something

File: Requirements.md (hunk 4 above)
Comment (bedrin) [reply Requirements.md]: Other examples of NFR would be supported browsers (mobile?), concurrent users, availability, etc.

**Review by bedrin — CHANGES_REQUESTED** [review]
Great spec. I would change a few bits - see my comments inline.

Probably can be renamed to README.md so it would show up on github when you open your project - as you wish

File: Requirements.md (hunk 1 above)
Comment (vladdord) [reply Requirements.md]: Removed and changed non-functional requirements

File: Requirements.md (hunk 2 above)
Comment (vladdord) [reply Requirements.md]: Done

File: Requirements.md (hunk 3 above)
Comment (vladdord) [reply Requirements.md]: Removed

File: Requirements.md (hunk 4 above)
Comment (vladdord) [reply Requirements.md]: Numbers are very blurry and now close to random. We need to implement it first and then test performance

File: Requirements.md (hunk 5 above)
Comment (vladdord) [reply Requirements.md]: Done

**Comment by bedrin** [comment]
Great! Let's merge and I'll ask our FA's to review properly.

**Pull request merged**